index.html

```diff
@@ -3,9 +3,33 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+  <link rel="stylesheet" href="./css/style.css">
   <title>Document</title>
 </head>
 <body>
-  
+  <div class="container">
+    <h1 class="text-center">Library</h1>
+    <div class="row">
+      <div class="col-5 mx-auto">
+        <div id="errors" class="bg-info text-light"></div>
+        <form class="text-center" id="searchBook">
+          <label for="title">Title</label>
+          <input class="form-control mb-3" type="text" name="book[title]" id="title">
+          <label for="author">Author</label>
+          <input class="form-control mb-3" type="text" name="book[author]" id="author">
+          <label for="pages">Pages</label>
+          <input class="form-control mb-3" type="number" name="book[pages]" id="pages">
+          <label for="status">Aleady read</label>
+          <input class="form-control mb-3" type="checkbox" name="book[status]" id="status">
+          <input class="btn btn-primary btn-block btn-lg" type="submit" value="search">
```

Comment:
should `value` not be instead `Add Book` ???

Reply:
hahaha, we noticed but forgot to changed before we sent it to review again, lemme change it, and then merge, thanks for the observation :D